### PR TITLE
Testing creative bulma

### PR DESCRIPTION
**Pull request by kalwalt**
Testing the new creative-bulma v.1.0.3 - thanks to @CreativeBulma - on a Mobile menu with accordions. A rivisited version of the code 
of this PR https://github.com/kalwalt/gatsby-starter-i18n-bulma/pull/93
Read the comments in the code for more details.
The code is compiled fine in both development and build phase.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const Header = class extends React.Component {
        console.log(bulmaCollapsibleInstance.collapsed());
    });
 
-   const bulmaCollapsibleElement_1 = new bulmaCollapsible('#collapsible-message-accordion-1');
+  /* const bulmaCollapsibleElement_1 = new bulmaCollapsible('#collapsible-message-accordion-1');
```
Comment (kalwalt) [inline src/components/Header.js]: excluding this piece of code, make i t works. I dont receive anymore this error:

```javascript
index.js:2177 TypeError: Cannot read property 'container' of undefined
    at new Component (component.js:10)
    at new bulmaCollapsible (index.js:9)
    at Header.componentDidMount (Header.js:66)
    at Header.componentDidMount (react-hot-loader.development.js:654)
    at commitLifeCycles (react-dom.development.js:17335)
    at commitAllLifeCycles (react-dom.development.js:18737)
    at HTMLUnknownElement.callCallback (react-dom.development.js:150)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:200)
    at invokeGuardedCallback (react-dom.development.js:257)
    at commitRoot (react-dom.development.js:18949)
    at react-dom.development.js:20419
    at Object.unstable_runWithPriority (scheduler.development.js:255)
    at completeRoot (react-dom.development.js:20418)
    at performWorkOnRoot (react-dom.development.js:20347)
    at performWork (react-dom.development.js:20255)
    at performSyncWork (react-dom.development.js:20229)
    at requestWork (react-dom.development.js:20098)
    at scheduleWork (react-dom.development.js:19912)
    at scheduleRootUpdate (react-dom.development.js:20573)
    at updateContainerAtExpirationTime (react-dom.development.js:20601)
    at updateContainer (react-dom.development.js:20658)
    at ReactRoot../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js.ReactRoot.render (react-dom.development.js:20954)
    at react-dom.development.js:21091
    at unbatchedUpdates (react-dom.development.js:20460)
    at legacyRenderSubtreeIntoContainer (react-dom.development.js:21087)
    at render (react-dom.development.js:21199)
    at app.js:62
__stack_frame_overlay_proxy_console__ @ index.js:2177
error @ react-hot-loader.development.js:244
componentDidCatch @ react-hot-loader.development.js:2160
callback @ react-dom.development.js:18096
callCallback @ react-dom.development.js:16434
commitUpdateEffects @ react-dom.development.js:16473
commitUpdateQueue @ react-dom.development.js:16461
commitLifeCycles @ react-dom.development.js:17365
commitAllLifeCycles @ react-dom.development.js:18737
callCallback @ react-dom.development.js:150
invokeGuardedCallbackDev @ react-dom.development.js:200
invokeGuardedCallback @ react-dom.development.js:257
commitRoot @ react-dom.development.js:18949
(anonymous) @ react-dom.development.js:20419
unstable_runWithPriority @ scheduler.development.js:255
completeRoot @ react-dom.development.js:20418
performWorkOnRoot @ react-dom.development.js:20347
performWork @ react-dom.development.js:20255
performSyncWork @ react-dom.development.js:20229
requestWork @ react-dom.development.js:20098
scheduleWork @ react-dom.development.js:19912
scheduleRootUpdate @ react-dom.development.js:20573
updateContainerAtExpirationTime @ react-dom.development.js:20601
updateContainer @ react-dom.development.js:20658
./node_modules/@hot-loader/react-dom/cjs/react-dom.development.js.ReactRoot.render @ react-dom.development.js:20954
(anonymous) @ react-dom.development.js:21091
unbatchedUpdates @ react-dom.development.js:20460
legacyRenderSubtreeIntoContainer @ react-dom.development.js:21087
render @ react-dom.development.js:21199
(anonymous) @ app.js:62
setTimeout (async)
(anonymous) @ ready.js:32
(anonymous) @ app.js:61
Promise.then (async)
(anonymous) @ app.js:54
Promise.then (async)
./.cache/app.js @ app.js:16
__webpack_require__ @ bootstrap:766
fn @ bootstrap:129
0 @ tags.js:98
__webpack_require__ @ bootstrap:766
(anonymous) @ bootstrap:903
(anonymous) @ bootstrap:903
index.js:2177 The above error occurred in the <AppContainer> component:
    in AppContainer (created by HotExportedHomePage)
    in HotExportedHomePage (created by PageRenderer)
    in PageRenderer (created by JSONStore)
    in JSONStore (created by EnsureResources)
    in ScrollContext (created by EnsureResources)
    in RouteUpdates (created by EnsureResources)
    in EnsureResources (created by RouteHandler)
    in RouteHandler (created by Root)
    in div (created by FocusHandlerImpl)
    in FocusHandlerImpl (created by Context.Consumer)
    in FocusHandler (created by RouterImpl)
    in RouterImpl (created by LocationProvider)
    in LocationProvider (created by Context.Consumer)
    in Location (created by Context.Consumer)
    in Router (created by Root)
    in Root
    in _default
```

React will try to recreate this component tree from scratch using the error boundary you provided, LocationProvider.

Probably it is not needed, and it is needed only this:

```js

   const bulmaCollapsibleInstances = bulmaCollapsible.attach('.is-collapsible');

```

File: src/components/Header.js [hunk 2]
```diff
@@ -53,22 +54,54 @@ const Header = class extends React.Component {
...
-    nM.style.backgroundColor = "#abd6d1"
-    // Get all "accordions" elements
-    const accordion = Array.prototype.slice.call(document.querySelectorAll('.accordions'), 0);
-    // Check if there are any navbar links
-    if (accordion.length > 0) {
+    // Return an array of bulmaCollapsible instances (empty if no DOM node found)
+   const bulmaCollapsibleInstances = bulmaCollapsible.attach('.is-collapsible');
+
+   // Loop into instances
+   bulmaCollapsibleInstances.forEach(bulmaCollapsibleInstance => {
```
Comment (kalwalt) [inline src/components/Header.js]: i think that i don't need this, i added only for testing.

File: src/components/Header.js [hunk 3]
```diff
@@ -53,22 +54,54 @@ const Header = class extends React.Component {
...
   if (isMobile) {
 
-    let nM = document.getElementById("navMenu");
-    nM.style.backgroundColor = "#abd6d1"
-    // Get all "accordions" elements
-    const accordion = Array.prototype.slice.call(document.querySelectorAll('.accordions'), 0);
-    // Check if there are any navbar links
-    if (accordion.length > 0) {
+    // Return an array of bulmaCollapsible instances (empty if no DOM node found)
+   const bulmaCollapsibleInstances = bulmaCollapsible.attach('.is-collapsible');
```
Comment (kalwalt) [inline src/components/Header.js]: @CreativeBulma For a minimal configuration, does it is enough to open and close an accordion?

Comment (CreativeBulma) [reply src/components/Header.js]: Yes it's the only code you need for basic behavior.
If you don't plan to manipulate instances later you don't even need to save them into a const.
If you want an element to be opened by default just add <code>is-active</code> class on it.

Comment (kalwalt) [reply src/components/Header.js]: @CreativeBulma thank you, i think i will use the simplest one. Anyway i will do other tests.

**Comment by kalwalt** [comment]
I will try for the [infoCard Component](https://github.com/kalwalt/gatsby-starter-i18n-bulma/blob/dev/src/components/InfoCard.js) in another branch and i will merge this.

**Pull request merged**